web/github.go

```diff
@@ -46,8 +63,52 @@ func (g *GitHub) installations(ctx context.Context) ([]*github.Installation, err
 	return installations, nil
 }
 
-// orgResponse is the response type used by Orgs handler
-type orgRespose struct {
+var errNotFound = errors.New("not found")
+
+// installation returns the GitHub App Installation corresponding to the given
+// organization name. If the installation is not found the error is errNotFound
+func (g *GitHub) installation(ctx context.Context, orgName string) (*github.Installation, error) {
+	appClient, err := g.appClient()
+	if err != nil {
+		return nil, err
+	}
+
+	installation, _, err := appClient.Apps.FindOrganizationInstallation(ctx, orgName)
+	if err != nil {
+		if err.(*github.ErrorResponse).Response.StatusCode == http.StatusNotFound {
+			return nil, errNotFound
+		}
+
+		return nil, err
+	}
+
+	return installation, nil
+}
+
+func (g *GitHub) admin(w http.ResponseWriter, r *http.Request, installation *github.Installation, login string) (bool, error) {
```

Comment:
since `admin` returns a boolean, I wonder if it would be better `isAdmin`
```suggestion
func (g *GitHub) isAdmin(w http.ResponseWriter, r *http.Request, installation *github.Installation, login string) (bool, error) {
```